examples/errors.js

```diff
@@ -34,8 +31,8 @@ const warning = function() {
 
 // Emit a `multipleResolves` event
 const multipleResolves = function() {
-  // eslint-disable-next-line no-new, promise/avoid-new
-  new Promise((resolve, reject) => {
+  // eslint-disable-next-line promise/avoid-new
+  return new Promise((resolve, reject) => {
```

Comment:
Same here: no need to return the promise because that function is not meant to return a promise, it's meant to create a multipleResolves process event. Now I think returning it might still make the function look more natural, so it's a good idea.